src/worker.py

```
@@ -1416,3 +1443,125 @@ async def on_fetch(request, env):
+
+    notifications = []
+    for r in rows.results or []:
+        notifications.append({
+            "id":         r["id"],
+            "type":       r["type"],
+            "title":      r["title"],
+            "message":    r["message"],
+            "is_read":    bool(r["is_read"]),
+            "related_id": r["related_id"],
+            "created_at": r["created_at"],
+        })
```

Ruff (0.15.10)

[warning] Use a list comprehension to create a transformed list

(PERF401)